packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -160,16 +155,14 @@ export class VaTextInput {
     i18next.on('languageChanged', () => {
       forceUpdate(this.el);
     });
-  };
+  }
 
   disconnectedCallback() {
     i18next.off('languageChanged');
-  };
+  }
 
   render() {
-    const describedBy =
-      `${this.ariaDescribedby} ${this.error ? 'error-message' : ''}`.trim() ||
-      null; // Null so we don't add the attribute if we have an empty string
+    const describedBy = `${this.error ? 'error-message' : ''}`.trim() || null; // Null so we don't add the attribute if we have an empty string
```

Comment:
Now that we don't have the chance of `my-custom-describedby-id `  with the dangling space for when `this.error` is undefined, I don't think we need to `.trim()`.
```

Reply:
PR has been updated and `.trim()` removed

Reply:
I think this could be cleaned up a bit more to remove some things that are no longer necessary after removing the `ariaDescribedby` prop.

Reply:
Ok what else do you want me to remove?

Reply:
If we're not using that aria-describedby prop to construct a string, do we need the ternary with the empty string here?

Reply:
@k80bowman the code is currently the following `const describedBy = `${this.error ? 'error-message' : ''}` || null;`
Which is just saying if someone sets an error message add the attribute of `aria-describedby='error-message'` to the input element otherwise if it is null don't set this attribute. By setting the attribute it will then correspond with this line via the id https://github.com/department-of-veterans-affairs/component-library/blob/f7aabf936611d78338ff35fb649553437449ba8c/packages/web-components/src/components/va-text-input/va-text-input.tsx#L180 and will then read out the error message to the user is focused on the input field

Reply:
Oh I see what you are getting at I will update it to `const describedBy = ${this.error ? 'error-message' : null};`

Reply:
Just updated the PR with that fix

Reply:
So I wrote my own test to check this and it does work, but would you mind please adding a test to check that this does not add an aria-describedby when an empty string is passed as the error message prop?